decentraland/unity-renderer
# chore: different POIs improvements

## What does this PR change?

Fix #1901
Removes default name POIs, reduces number of characters of title to avoid more than 2 lines of text, centers the POI icon in the square of the parcel

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/poi-improvements
2. Verify in the map the changes listed above

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
